docs/howto/upgrade_from_0_8_to_0_10.md

```diff
@@ -0,0 +1,227 @@
+[Back to Guides](../README.md)
+
+# How to migrate from `0.8` to `0.10` safely
+
+## Disclaimer
+### Proceed at your own risk
+This document attempts to outline steps to upgrade your app based on the collective experience of
+developers who have done this already. It may not cover all edge cases and situation that may cause issues, 
+so please proceed with a certain level of caution. 
+
+## Overview
+This document outlines the steps needed to migrate from `0.8` to `0.10`. The method described
+below has been created via the collective knowledge of contributions of those who have done
+the migration successfully. The method has been tested specifically for migrating from `0.8.3`
+to `0.10.2`.
+
```

Comment:
am thinking it might be good to start out with what the basic issues are

- passing a serializer to render
- passing nil to a serializer
- not having embed
- not defining attribute methods on the serializer
- collection root
- no default serializer
- serializer `instance_options` vs `@options`.
- private vs. protected methods?